content/ending-frozen-applications.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Ending Frozen Applications and Desktops
+description: >
+  Linux provides several methods to kill frozen or hung up processes and applications using GUI applications or terminal commands.
+keywords:
+  - frozen applications
+  - frozen desktop
+  - frozen terminal
+  - hung up
+  - not responding
+  - System76
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section:
+---
+
+## Ending Frozen Processes, Applications, and Desktops
+
+Linux offers many methods to end hanging or frozen processes. The `gnome-system-monitor` provides a GUI process management application with features similar to the Windows Task Manager, while the `kill` command can end single or multiple processes from the Terminal.
+
+>Note: This article details several methods to end processes for a given scenario, but is not exhaustive or guaranteed to provide the most appropriate method for every scenario. Users should also keep in mind that suddenly terminating processes can result in unwanted data loss or system stability issues.
+
+## End Specific Applications or Windows
+
+### Using gnome-system-monitor (GUI)
+
+The gnome-system-monitor comes prepackaged in Pop!_OS. This application provides a GUI interface and functionality similar to the Windows Task Manager.
+
+1. Press <kbd>Super</kbd> + <kbd>T</kbd> to launch a Terminal session.
+2. Enter `gnome-system-monitor` and hit <kbd>Enter</kbd>.
+3. Locate frozen and hanging applications.
+
+   ![gnome-system-monitor](/images/ending-frozen-applications/gnome-system-monitor)
+
+4. Right click the application, click Stop in the dropdown menu.
```

Comment:
Sounds like a word is missing after the comma; I would either add the word "then" after the comma, or remove the comma and use the word "and" instead, whichever you prefer.

Unrelated to the above, I wonder if the End (`SIGTERM`) or Kill (`SIGKILL`) options ever work when Stop (`SIGSTOP`) doesn't. I know there have been times when I needed to add a `-9` to a `kill` command for it to work, which is why I would think `Kill` might be needed in some cases. (Could be worth advising to try `Stop` first, then `End`, then `Kill` if you're trying to avoid having users jump straight to the most destructive option?)

Reply:
according to kill's manpage it uses "TERM" (end) as the default, so this particular section would not be using the destructive -9. The section using TTY is sort of hail-mary for a completely frozen system. If -9 is bad practice I can add a suggestion to use the default (15). SIGSTOP I'm not familiar with, but appears to simply pause a running process. I'm not sure if that's useful for completely locked-up processes, I'll defer to your expertise.

Reply:
I have implemented these changes - thanks!